datastore/db/models.py

```diff
@@ -318,8 +318,12 @@ def __str__(self):
         return "%s (datagetter %s)" % (self.name, str(self.getter_run.datetime))
 
     class Meta:
-        unique_together = ("getter_run", "prefix")
         ordering = ["prefix"]
+        constraints = [
+            models.UniqueConstraint(
+                fields=["org_id", "getter_run"], name="publisher_unique_org_id"
```

Comment:
This doesn't quite do the same thing that it's replacing. The `prefix` field is a threesixtygiving internal publisher code which doesn't have the same purpose as the `org_id` which is an external code and not necessarily controlled by 360Giving, e.g. if a publisher charity number changed the org-id would change but the publisher prefix won't.

However the main purpose of the constraint is to cause an error if duplicated data was loaded so I don't think this is necessarily an unwanted change (we only recently added org-ids to publisher data (which comes from the registry)) so we need to check whether they consider this a stable identifier. 

If we keep this constraint the test data needs to be updated as this is currently causing it to fail CI https://github.com/ThreeSixtyGiving/datastore/actions/runs/6383574480/job/17324468838#step:14:219

Reply:
The other thing this is connected to is https://github.com/ThreeSixtyGiving/datastore/issues/129 which is based on a similar question of whether we maintain publisher data across multiple datagetter runs.